src/hooks/posters.ts

```diff
@@ -71,24 +71,53 @@ export const usePosters = (posterHallId: string) => {
   const fuseVenues = useMemo(
     () =>
       new Fuse(filteredPosterVenues, {
-        keys: ["poster.title", "poster.authorName", "poster.categories"],
+        keys: [
+          "name",
+          "poster.title",
+          "poster.authorName",
+          "poster.categories",
+        ],
+        threshold: 0.2, // 0.1 seems to be exact, default 0.6: brings too distant if anyhow related hits
+        ignoreLocation: true, // default False: True - to search ignoring location of the words.
+        findAllMatches: true,
       }),
     [filteredPosterVenues]
   );
 
   const searchedPosterVenues = useMemo(() => {
-    if (!searchQuery)
-      return filteredPosterVenues.slice(0, displayedPostersCount);
+    const normalizedSearchQuery = searchQuery.trim();
+
+    if (!normalizedSearchQuery) return filteredPosterVenues;
 
     return fuseVenues
-      .search(searchQuery)
-      .slice(0, displayedPostersCount)
+      .search({
+        //@ts-ignore
+        $and: normalizedSearchQuery
+          .match(/("[^"]*?"|[^"\s]+)+(?=\s*|\s*$)/g) // source: https://stackoverflow.com/a/16261693/1265472
+          .map((x) => ({
+            $or: [
+              { name: x },
+              { "poster.title": x },
+              { "poster.authorName": x },
+              { "poster.categories": x },
+            ],
+          })),
+      })
       .map((fuseSearchItem) => fuseSearchItem.item);
-  }, [searchQuery, fuseVenues, filteredPosterVenues, displayedPostersCount]);
+  }, [searchQuery, fuseVenues, filteredPosterVenues]);
+
+  const displayedPosterVenues = useMemo(
+    () => searchedPosterVenues.slice(0, displayedPostersCount),
+    [searchedPosterVenues, displayedPostersCount]
+  );
+
+  const hasHiddenPosters =
+    searchedPosterVenues.length > displayedPosterVenues.length;
```

Comment:
`searchedPosterVenues` is returning `filteredPosterVenues`  When `(!normalizedSearchQuery)`. 
After that `filteredPosterVenues` seems return `posterVenues` which is using this selector `posterVenuesSelector`. Looks like it can be undefined and this `.length` can fail. 

Maybe you should have `const posterVenues = useSelector(posterVenuesSelector) ?? []` in `usePosterVenues`

Reply:
Nevermind, I just saw this: https://github.com/sparkletown/sparkle/pull/1460/files#diff-06ebbfcf8e627f41bd6b15a2648431dc8a5dfd21f7d64f9bc798ed55051c0c58R35

You can ignore this comment and resolve the conversation.